readme.md

```diff
@@ -24,13 +24,13 @@ You've written the following HTML. When you look at it in your browser, it's jus
 </html>
 ```
 
-> Your answer...
+> The `<title>` element has no closing tag.
```

Comment:
My hope is that they'll think to just run the HTML through a validator, and problem solved